eng/scripts/Update-PackageVersion.ps1

```diff
@@ -0,0 +1,71 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/main/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER ServiceDirectory
+The Name of the Service Directory
+
+.PARAMETER PackageName
+The Name of the Package
+
+.PARAMETER NewVersionString
+Use this to overide version incement logic and set a version specified by this parameter
+
+.EXAMPLE
+Updating package version for Azure.Core
+Update-PackageVersion.ps1 -ServiceDirectory core -PackageName azure_core
+
+Updating package version for Azure.Core with a specified verion
+Update-PackageVersion.ps1 -ServiceDirectory core -PackageName azure_core -NewVersionString 2.0.5
+
+Updating package version for Azure.Core with a specified verion and release date
+Update-PackageVersion.ps1 -ServiceDirectory core -PackageName azure_core -NewVersionString 2.0.5 -ReleaseDate "2020-05-01"
+#>
+
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory = $True)]
+  [string] $ServiceDirectory,
+  [Parameter(Mandatory = $True)]
+  [string] $PackageName,
+  [string] $NewVersionString,
+  [string] $ReleaseDate,
+  [boolean] $ReplaceLatestEntryTitle = $true
+)
+
+. (Join-Path $PSScriptRoot '../common/scripts/common.ps1')
+
+$pkgProperties = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $ServiceDirectory
+$packageVersion = $pkgProperties.Version
+
+$packageSemVer = [AzureEngSemanticVersion]::new($packageVersion)
+Write-Host "Current Version: ${PackageVersion}"
+
+if ([System.String]::IsNullOrEmpty($NewVersionString)) {
+  $packageSemVer.IncrementAndSetToPrerelease();
```

Comment:
For 0.x releases, does this set the minor? If not, until we GA would it be worth the effort?

Reply:
0.x releases are already considered "Prerelease" so they get a minor bump by default.  i.e.
The next prerelease version for "0.3.0" is "0.4.0".
The next prerelease version for "1.3.1" is "1.4.0-beta.1".
The next prerelease version for "1.4.0-beta.1" is "1.4.0-beta.2"

Reply:
I added this to eng/common a few weeks ago:
https://github.com/Azure/azure-sdk-tools/pull/9668